python/Ganga/Utility/Shell.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, setup=None, setup_args=[]):
             command = 'source %s %s > /dev/null 2>&1; python -c "from __future__ import print_function; import os; print(os.environ)"' % (setup, " ".join(setup_args))
             logger.debug('Running:   %s' % command )
             pipe = subprocess.Popen('bash', env=env, cwd=this_cwd, shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
-            output = pipe.communicate(command)
+            output = pipe.communicate(command.encode())
```

Comment:
Yes, as `command` is a unicode string and `communicate()` expects a locale-encoded byte stream (almost always `utf-8`).